rpc_jobs/multi_node_aio.yml

```diff
@@ -101,7 +101,11 @@
       # Leapfrog upgrades are only run for kilo --> newton141
       # as the upgrade method is not supported for any
       # other target series.
-      # the kilo --> newton141 job is triggered by PRs to either branch.
+      # the kilo --> newton141 job is only ran as a periodic job until
+      # it's more reliable and can be tested as a PR
+      - series: kilo
+        action: leapfrogupgrade
+        trigger: pr
```

Comment:
How about:

```
- action: leapfrogupgrade
   trigger: pr
```

Reply:
K, updated with the two lines as it sounds like using three entries in an exclude doesn't work.